lib/wrapper.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Wrapper for package installation, zypper or transactional update
+# Maintainer: QE Core <qe-core@suse.de>
+
+package wrapper;
+use Mojo::Base qw(Exporter);
```

Comment:
In the future, we can take a look into using perl signatures https://www.perl.com/article/announcing-perl-7/

Reply:
Sure